source/features/set-default-repositories-type-to-sources.tsx

```diff
@@ -1,25 +1,35 @@
 import select from 'select-dom';
+import {observe} from 'selector-observer';
 
 import features from '.';
 
+function addSourceTypeToLink(link: HTMLAnchorElement): void {
+	const search = new URLSearchParams(link.search);
+	search.set('type', 'source');
+	link.search = String(search);
+}
+
 function init(): void {
 	const links = select.all<HTMLAnchorElement>([
 		// Pre "Repository refresh" layout
-		'.user-profile-nav [href$="tab=repositories"]', // "Your repositories" in header dropdown
 		'#user-links [href$="tab=repositories"]', // "Repositories" tab on user profile
 		'.orgnav > .pagehead-tabs-item:first-child', // "Repositories" tab on organization profile
 		// "Repository refresh" layout
-		'.header-nav-current-user ~ [href$="tab=repositories"]', // "Your repositories" in header dropdown
 		'[aria-label="User profile"] [href$="tab=repositories"]', // "Repositories" tab on user profile
 		'[aria-label="Organization"] .UnderlineNav-item:first-child', // "Repositories" tab on organization profile
 		'[data-hovercard-type="organization"]' // Organization name on repo header + organization list on user profile
 	]);
 
 	for (const link of links) {
-		const search = new URLSearchParams(link.search);
-		search.set('type', 'source');
-		link.search = String(search);
+		addSourceTypeToLink(link);
 	}
+
+	observe('.header-nav-current-user ~ [href$="tab=repositories"]', { // "Your repositories" in header dropdown
```

Comment:
We should copy this instead:

https://github.com/sindresorhus/refined-github/blob/1bedcd71886ac9c20bf91a6d48e428622ab6d1b1/source/features/deprioritize-marketplace-link.tsx#L22-L24

`observe` is too heavy for a single change that might never happen